docs/docs/components/quickstart/nodejs/actions/README.md

```diff
@@ -389,6 +389,41 @@ export default {
 }
```

Comment:
@dylburger lmk what you think of these changes

Reply:
looks great